tests/integration/ethernet_mtu_test.py

```diff
@@ -77,6 +85,8 @@ def test_decrease_to_zero_iface_mtu():
     eth1_desired_state = desired_state[INTERFACES][0]
     eth1_desired_state['state'] = 'up'
     eth1_desired_state['mtu'] = 0
+    eth1_desired_state['ipv6']['enabled'] = True
+    # MTU set failure will revert back to 1500 MTU where IPv6 will be enabled.
```

Comment:
I do not understand the comment. It reverts to the original value (default or not).
If nothing changed from the original test, I would drop these comments (line 104, 120 as well).